app/views/houses/index.html.slim

```diff
@@ -0,0 +1,46 @@
+javascript:
+  $(function () {
+    $('[data-toggle="tooltip"]').tooltip()
+  })
+
+div.house-container
+
+  = form_tag houses_path, method: :get, class: "house-form" do
+    div.house-finder
+      a.btn.btn-primary  href="#{houses_path(:date => @date.prev_month.strftime('%Y-%m'))}"
+        | Prev Month 
+      div.house-date-picker
+        b DAD Houses for:
+        = select_date(@date, discard_day: true, start_year: 2020, end_year: 2030)
+        = submit_tag 'Select', class: "btn btn-primary"
+      a.btn.btn-primary href="#{houses_path(:date => @date.next_month.strftime('%Y-%m'))}"
+        | Next Month
+
+  div.house-neighborhood
+    - if @houses.any?
+      - @houses.each_with_index do |h, i|
+        div class="card house-card"
+          .card-header.house-card-header
+            b style="flex: 1" #{h.house_name}  
+            | #{"🥇" if h.place == 'first'} #{"🥈" if h.place == 'second'} #{"🥉" if h.place == 'third'}
```

Comment:
Emoji's are part of unicode and read just like normal text. Most modern operating systems support them either natively or with a package.
https://en.wikipedia.org/wiki/Emoji#Modern-day_emojis_(2010%E2%80%93present)

I would have used an icon, but font awesome doesn't have a first second and third trophy/medal.

I'll take it out if you don't like it. I figured since it was just HTML it wouldn't be a big deal and this seemed like the simplest solution. (I kinda like it to be honest 😂)